test/unit/attribute/date_time_test.rb

```diff
@@ -21,7 +21,7 @@ class Aggregate::Attribute::DateTimeTest < ActiveSupport::TestCase
     time = Time.at(1_544_732_833).in_time_zone("Pacific Time (US & Canada)")
```

Comment:
Would it be worthwhile to include milliseconds when creating the timestamp?

Reply:
+1 to Bobby's comment ^

Reply:
Done.